src/pages/Page.stories.ts

```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { within, userEvent } from '@storybook/testing-library';
+// import { within, userEvent } from '@storybook/testing-library';
```

Comment:
removed references to the old storybook. This page will be rewritten and is only here for proof of concept.

Reply:
Hey @storybook/testing-library will be useful in the later stages of a project without a lot of technical debt. I suggest removing this library completely and using those libraries for unit tests
@testing-library/user-event
@testing-library/react

Comments need to be deleted

Reply:
As I said, this page is only here for conceptual purposes and will eventually be totally overwritten. I appreciate the input on  the storybook testing-library but it clashed with existing testing framework and until I have had the time to look into storybooks testing capabilities and how it affects our plans for our CI/CD pipeline I am removing.